extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -31,18 +41,24 @@ const TrackIcon = ( { isPlaying, isError, className } ) => {
 		return <span className={ className } />;
 	}
 
-	return <span className={ `${ className } ${ className }--${ name }` }>{ icon }</span>;
+	return (
+		<span className={ `${ className } ${ className }--${ name }` }>
+			<span className="jetpack-podcast-player--visually-hidden">{ hiddenText }</span>
+			{ icon }
+		</span>
+	);
 };
 
 import { getColorClassName } from '../utils';
 
-const TrackError = memo( ( { link } ) => (
+const TrackError = memo( ( { link, title } ) => (
 	<div className="jetpack-podcast-player__track-error">
 		{ __( 'Episode unavailable', 'jetpack' ) }{ ' ' }
 		{ link && (
 			<span>
 				{ '(' }
 				<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
+					<span class="jetpack-podcast-player--visually-hidden">{ title }: </span>
 					{ __( 'Open in a new tab', 'jetpack' ) }
 				</a>
 				{ ')' }
```

Comment:
@akirk Would you mind chiming in on this? I wonder if the semi-colon and the parenthesis should be translatable or not. In an RTL language, would the semi-colon be after the title string, or should it be included in the string to be easier to translate?

Reply:
I also see untranslated parentheses, I think this should be completely restructured.

Reply:
The parenthesis are outside of the `<a>` in order to style it per design. We tried several different routes of getting it inside the `<a>` and still meet the style, but nothing worked well (pseudo elements, etc). 

Are we able to include HTML inside of a translated string? The point of the added `<span class="jetpack-podcast-player--visually-hidden">` is to give more context to the link for the VoiceOver Link Rotor (and for screen readers in general). If we're not able to put HTML inside of `__()`, then I'm not sure how we'd go about this.

```
<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
	{ sprintf( __( '<span class="jetpack-podcast-player--visually-hidden">%s: </span><span class="remove-underline">(</span>Open in a new tab<span class="remove-underline">)</span>', 'jetpack' ), title )  }
</a>
```

When I try the above code, I get this. Same if I put some of the HTML in the `sprintf()` with `title` as an experiment.
<img width="735" alt="Screen Shot 2020-04-01 at 8 37 10 AM" src="https://user-images.githubusercontent.com/967608/78143645-2b117400-73f4-11ea-91e7-b17368cebb6d.png">

cc @marekhrabe @WunderBart @davemart-in

Reply:
The stringified HTML won't work, unless we put it into `dangerouslySetInnerHTML`. I'd like to avoid that. How can we simplify the message. Can we get rid of those parentheses, @davemart-in?

Reply:
Sure. Is something like this feasible?

<img width="283" alt="Image 2020-04-01 at 10 19 30 AM" src="https://user-images.githubusercontent.com/5634774/78148046-5bf8a580-7402-11ea-9f57-4e620780ec35.png">

Reply:
I've updated the code to match the design with the `-`. I also made the visually hidden title into a `sprintf`.

<img width="752" alt="Screen Shot 2020-04-01 at 9 39 56 AM" src="https://user-images.githubusercontent.com/967608/78150561-11c0f580-73fd-11ea-80f6-70d2d30b88ee.png">

Reply:
I merged this after updating it as much as I could, as this seemed easier to iterate on outside of this PR. If this is still an issue or should get changed, I can add it as an issue and reference it in in the master `feature/podcast-player` branch. Just let me know how you'd like me to proceed @jeherve and @akirk. Thanks for the review and help!